src/main/scala/org/broadinstitute/dsde/rawls/workspace/WorkspaceService.scala

```diff
@@ -6,12 +6,15 @@ import org.broadinstitute.dsde.rawls.model.Workspace
 import org.broadinstitute.dsde.rawls.workspace.WorkspaceService._
 import org.broadinstitute.dsde.rawls.ws.PerRequest
 import spray.http.StatusCodes
+import spray.httpx.SprayJsonSupport._
+import org.broadinstitute.dsde.rawls.model.WorkspaceJsonSupport._
 
 /**
  * Created by dvoet on 4/27/15.
  */
 object WorkspaceService {
   case class SaveWorkspace(workspace: Workspace)
```

Comment:
I like defining a sealed trait to define the concept of "message for this actor" and then extending that w/ all my message class/objects

    sealed trait WorkspaceServiceMessage
    case class SaveWorkspace(workspace: Workspace) extends WorkspaceServiceMessage

It makes it clear that I'm looking at an actor message